app/helpers/message_thread_helper.rb

```diff
@@ -0,0 +1,9 @@
+module MessageThreadHelper
+  def self.show_recipient?(message_thread)
+    message_thread.box.communication_to_multiple_subjects? && message_thread.is_outbox && message_thread.recipient.present?
```

Comment:
Tomuto `comunication_to_multiple_subjects?` inak uplne nerozumiem. Co to znamena?

Reply:
Snazila som sa tym pomenovat to, ci ten dany typ boxu komunikuje k/od X subjektom - rozne institucie (ako Upvs::Box), vtedy ma zmysel rozlisovat pripady kedy zobrazujeme prijimatela/odosielatela alebo
to je komunikacia vzdy voci 1 subjektu (ako Fs::Box) a vtedy ma zmysel vzdy ukazovat iba odosielatela, resp. asi skor nazov boxu, ktoreho sa to tyka.

Reply:
Aha, dava zmysel. V tom pripade mi dava skor zmysel to otocit `single_recipient?`

Reply:
Upravila som